$NetBSD: patch-aa,v 1.11 2010/07/29 15:49:36 joerg Exp $

Do not fall through to the implicit prototype as it conflicts with
stdlib.h.

--- lib/autoconf/functions.m4.orig	2010-07-29 15:21:52.000000000 +0000
+++ lib/autoconf/functions.m4
@@ -369,6 +369,8 @@ AC_CACHE_CHECK([for alloca], ac_cv_func_
 # ifdef _MSC_VER
 #  include <malloc.h>
 #  define alloca _alloca
+# elif defined(__NetBSD__) || defined(__FreeBSD__) || defined(__DragonFly__) || defined(__OpenBSD__)
+#   include <stdlib.h>
 # else
 #  ifdef HAVE_ALLOCA_H
 #   include <alloca.h>
